webpack/dev.babel.js

```diff
@@ -24,32 +29,96 @@ const devConfig = configBuilder(
         "./src/standalone/index.js",
       ],
       "swagger-ui": "./src/style/main.scss",
+      vendors: ["react", "react-dom", "react-refresh/runtime"],
```

Comment:
This can be limited to following as `swagger-ui-bundle` already contains bundled `react` and `react-dom`.

```suggestion
      vendors: ["react-refresh/runtime"],
```

Reply:
fixed.